src/index.js

```diff
@@ -921,46 +1115,46 @@ export class ULabel {
         console.warn(`
             Passing in each argument as a seperate parameter to ULabel is now deprecated \n
             Please pass in an object with keyword arguments instead
-        `)
+        `);
 
         return {
-            "container_id": arguments[0],   // Required
-            "image_data": arguments[1],     // Required
-            "username": arguments[2],       // Required
-            "submit_buttons": arguments[3], // Required
-            "subtasks": arguments[4],       // Required
-            "task_meta": arguments[5] ?? null,       // Use default if optional argument is undefined
-            "annotation_meta": arguments[6] ?? null, // Use default if optional argument is undefined
-            "px_per_px": arguments[7] ?? 1,          // Use default if optional argument is undefined
-            "initial_crop": arguments[8] ?? null,    // Use default if optional argument is undefined
-            "initial_line_size": arguments[9] ?? null,  // Use default if optional argument is undefined
-            "config_data": arguments[10] ?? null,    // Use default if optional argument is undefined
-            "toolbox_order": arguments[11] ?? null   // Use default if optional argument is undefined
-        }
-    }
-
-    /** 
+            container_id: arguments[0], // Required
+            image_data: arguments[1], // Required
+            username: arguments[2], // Required
+            submit_buttons: arguments[3], // Required
+            subtasks: arguments[4], // Required
+            task_meta: arguments[5] ?? null, // Use default if optional argument is undefined
+            annotation_meta: arguments[6] ?? null, // Use default if optional argument is undefined
+            px_per_px: arguments[7] ?? 1, // Use default if optional argument is undefined
+            initial_crop: arguments[8] ?? null, // Use default if optional argument is undefined
+            initial_line_size: arguments[9] ?? null, // Use default if optional argument is undefined
+            config_data: arguments[10] ?? null, // Use default if optional argument is undefined
+            toolbox_order: arguments[11] ?? null, // Use default if optional argument is undefined
+        };
```

Comment:
Nice formatting got obliterated here. Should I add an ignore block, or something else?

Reply:
probably can add one `// Required` comment at the start of the list, and then a `// Use default if optional argument is undefined` comment right after the last required arg

Reply:
although i think i changed this anyways in #170